get_gcs_test.go

```diff
@@ -8,35 +8,20 @@ import (
 	"testing"
 )
 
-// initGCPCredentials writes a temporary GCS credentials file if necessary and
-// returns the path and a function to clean it up. allAuthenticatedUsers can
-// access go-getter-test with read only access.
-func initGCPCredentials(t *testing.T) func() {
-	if gc := os.Getenv("GOOGLE_CREDENTIALS"); gc != "" &&
-		os.Getenv("GOOGLE_APPLICATION_CREDENTIALS") == "" {
-		file, cleanup := tempFileContents(t, gc)
-		os.Setenv("GOOGLE_APPLICATION_CREDENTIALS", file)
-		return func() {
-			os.Setenv("GOOGLE_APPLICATION_CREDENTIALS", "")
-			cleanup()
-		}
-	}
-	return func() {}
-}
+//Note for external contributors: In order to run the GCS test suite, you will only be able to be run
```

Comment:
Small formatting nit.
```suggestion
// Note for external contributors: In order to run the GCS test suite, you will only be able to be run
```